api/payloads/service_offering.go

```diff
@@ -10,6 +10,25 @@ import (
 	jellidation "github.com/jellydator/validation"
 )
 
+type ServiceOfferingGet struct {
+	IncludeBrokerFields []string
```

Comment:
We should be using the `IncludeReourceRule` type as we [do](https://github.com/cloudfoundry/korifi/blob/df9925de6b12eeb51cc05d3dd70019250b8e0175/api/payloads/service_plan.go#L29) for plans